### DISCUSSION TITLE
Need speed limit function！

### DISCUSSION THREAD

**Discussion by tinyhare** [localsend/localsend#1827] [Discussion (English, International)]
In office environment I don't want to takes up all bandwidth that Influence the use of others.
And I don't want the network administrator disconnected my network！！
So speed limit need.

**Comment by JaredTamana** [comment]
It would be better to create a [feature request](https://github.com/localsend/localsend/issues/new?assignees=&labels=feature+%3Asparkles%3A&projects=&template=feature_request.yaml&title=Feature+request). This is a discussion/community support forum.
